py-rse/packaging.Rmd

```diff
@@ -508,26 +497,24 @@ if __name__ == '__main__':
     main()
 ```
 
-Now that we've defined out entry points
-and edited our command line programs so that `main` doesn't require any input arguments,
-let's remove our currently installed version of `zipf`
-and install it again with the updated setup file.
+Once we have made the corresponding change in `collate.py` and `plotcounts.py`,
+we can remove the currently-installed version of `zipf` and re-install it:
```

Comment:
Earlier we said: "The `-e` option indicates that we want to to install the package in 'editable' mode, which means that any changes we make in the package code are directly available to use without having to reinstall the package."

Does that mean we don't need to un-install and re-install here?